Rakefile

```diff
@@ -17,7 +17,7 @@ task :default => [:test]
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new(:test) do |t|
-  t.pattern = 'spec/{unit}/**/*.rb'
+  t.pattern = 'spec/{classes,defines,unit,functions,hosts,integration}/**/*_spec.rb'
```

Comment:
@glennsarti a better option may be to simply not use this `rake test` task and instead use the `rake spec` task defined by puppetlabs_spec_helper, e.g., https://github.com/puppetlabs/puppetlabs_spec_helper/blob/master/lib/puppetlabs_spec_helper/rake_tasks.rb#L320-L325. This declares the appropriate dependencies already, not sure we need to duplicate. Thoughts?